spec/services/storage_root_migration_service_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe StorageRootMigrationService do
+  let(:to_storage_root) { create(:moab_storage_root) }
+
+  let(:from_storage_root) { complete_moab1.moab_storage_root }
+
+  let(:complete_moab1) {
+    create(:complete_moab, status: 'ok',
+                           last_moab_validation: Time.now,
+                           last_checksum_validation: Time.now,
+                           last_archive_audit: Time.now)
+  }
+
+  let(:complete_moab2) { create(:complete_moab, status: 'invalid_moab', moab_storage_root: from_storage_root) }
+
+  let(:complete_moab3) { create(:complete_moab) }
+
+  it 'migrates the storage root' do
+    # Before migration
+    expect(complete_moab1.moab_storage_root).to eq(complete_moab2.moab_storage_root)
+    expect(complete_moab1.moab_storage_root).not_to eq(complete_moab3.moab_storage_root)
+
+    druids = described_class.new(from_storage_root.name, to_storage_root.name).migrate.map { |druid| druid }
```

Comment:
```suggestion
    druids = described_class.new(from_storage_root.name, to_storage_root.name).migrate
```

`#map` seems extraneous (and things still worked for me when running tests locally without it)

Reply:
Yeah, it was necessary in an earlier version in which I was returning an Enumerator.